src/controllers/trip.controller.js

```diff
@@ -216,10 +216,29 @@ class TripController {
    */
   static async getTripRequest(req, res) {
     const subjectID = req.params.tripRequestID;
-    const userID = req.user.id;
-    const data = await tripService.getTripRequest(subjectID, userID);
+    const data = await tripService.getTripRequest(subjectID);
     if (data) return response.successMessage(res, 'success', 200, data);
-    return response.errorMessage(res, 'Trip not found', 404);
+    return response.errorMessage(res, 'Trip not foundz', 404);
+  }
+
+  /** Function to search according to what the user is typing
+   * @param {object} req the request sent to the server
+   * @param {object} res the response returned
+   * @returns {object} found data
+   */
+  static async searchInTripRequestTable(req, res) {
```

Comment:
@jo-rdan No need to indicate the table you'll be searching from. I think search as a function name could be satisfactory. You can still provide more info (The tables it will fetch data from, expected parameters) about the function in its description.